lib/Daemonise/Plugin/Daemon.pm

```diff
@@ -1,11 +1,11 @@
 package Daemonise::Plugin::Daemon;
 
+use BSD::Resource qw( getrusage setrlimit RLIMIT_VMEM );
```

Comment:
I'm a bit wary of how this will port to the new servers (if we switch to ubuntu, but alas we can solve that later)